### PR TITLE
Added login page test

### PR DESCRIPTION
**Pull request by VolhaDzenisiuk**

### CODE REVIEW

**Review by EugenBorisik — COMMENTED** [review]
Could also add class with By variables for point 2 from task 30. I believe that you know them and it will take 5 minutes

**Comment by VolhaDzenisiuk** [comment]
@EugenBorisik please check src/test/java/LoginPage.java file, it contains By variables. Thank you

**Comment by EugenBorisik** [comment]
All of them were found using By.xpath(..) method. I also want to see other methods like By.id, By.cssSelector, etc.

**Review by EugenBorisik — APPROVED** [review]
Also remember about By.linkText, By.partialLinkText, By.tagName

**Comment by VolhaDzenisiuk** [comment]
@EugenBorisik please review task 40.

File: src/test/java/pages/AlertPage.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package pages;
+
+import org.openqa.selenium.By;
+
+public class AlertPage {
+
+    public static String URL = "https://the-internet.herokuapp.com/javascript_alerts";
+    public static By RESULT = By.xpath("//p[@id=\"result\"]");
+    public static By JS_ALERT = By.xpath("//div[@class=\"example\"]/ul/li[1]/button");
```
Comment (EugenBorisik) [inline src/test/java/pages/AlertPage.java]: Use onClick attribute for these 3 locators

File: src/test/java/tests/AlertTest.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        }
+    }
+
+    @Test
+    public void testJsPrompt() {
+        WebElement jsPrompt = driver.findElement(AlertPage.JS_PROMPT);
+        jsPrompt.click();
+        try {
+            Alert alert = driver.switchTo().alert();
+            alert.sendKeys("alert test");
```
Comment (EugenBorisik) [inline src/test/java/tests/AlertTest.java]: Move your text in variable and use it here and in assertion

File: src/test/java/tests/FrameTest.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    }
+
+    @After
+    public void closeBrowser() {
+        driver.close();
+    }
+
+    @Test
+    public void testFrame() {
+        ((JavascriptExecutor)driver).executeScript("tinyMCE.activeEditor.setContent('')");
```
Comment (EugenBorisik) [inline src/test/java/tests/FrameTest.java]: You can set text without JavascriptExecutor. Just switch to the frame and send keys to body

**Review by EugenBorisik — CHANGES_REQUESTED** [review]

**Review by EugenBorisik — APPROVED** [review]

**Pull request merged**